corehq/apps/dump_reload/management/commands/load_domain_data.py

```diff
@@ -17,14 +18,41 @@
 
 
 class Command(BaseCommand):
-    help = 'Loads data from the give file into the database.'
+    """This command expects a ZIP file containing one or more
+    gzip files and a 'meta.json' file containing doc counts for each
+    of the gzip files:
+
+    zip:
+       sql.gz
+       couch.gz
+       sql-other.gz
+       meta.json
+
+    The filenames of the gzip files must be formatted as <slug><suffix>.gz where
+        -  <slug> is one of 'sql', 'couch', 'domain', 'toggle'
+        -  <suffix> can be anything
+
+    meta.json:
+        Must contain a single JSON object with properties for each of the filnames
+        in the zip file. The value of the properties must be an integer count of
+        documents in the corresponding gzip file:
+
+            {"sql": 10, "couch": 12, "sql-other": 123}
```

Comment:
```suggestion
        Must contain a single JSON object with properties for each of the filnames
        in the zip file. The value of the properties must be a dict of
        document counts in the corresponding gzip file:

            {"domain": {"Domain": 1}, "sql": {"blobs.BlobMeta": 11, "auth.User": 1}, "couch": {}, "toggles": {}}
```

Or something like that, anyways - `meta.json` stores a nested dict:

https://github.com/dimagi/commcare-hq/blob/44e7e895c2bc79e083bafdea27ad4427e24180bb/corehq/apps/dump_reload/management/commands/dump_domain_data.py#L43-L43